README.md

```diff
@@ -101,7 +105,7 @@ What is the `ui-sref` directive, and how is it used?
 
 ```text
 Your answer goes here...
-```
+it creates a link that when clicked directs wihthout reloading the page```
 
```

Comment:
What you said is true, but fundamentally, it 'links' or references to another ui-router state.